mis_builder/models/mis_report_instance.py

```diff
@@ -235,9 +235,10 @@ def _compute_dates(self):
         domain=[('field_id.name', '=', 'debit'),
                 ('field_id.name', '=', 'credit'),
                 ('field_id.name', '=', 'account_id'),
-                ('field_id.name', '=', 'date')],
+                ('field_id.name', '=', 'date'),
+                ('field_id.name', '=', 'company_id')],
```

Comment:
Hopefully, this wont be a restriction - I think if a model does not have a company_id, then it needs to be re-visited in light of that.  Base Odoo generally gets it right...

Which of course, highlights, budgets will also need to be company aware...

Reply:
> if a model does not have a company_id, then it needs to be re-visited 

Indeed. I decided to document that as a breaking change, because making it optional would complicate the code for little benefit. It's easy to adapt such views, and there must not be many in the wild as 3.0 is still very young.

> budgets will also need to be company aware

A this point I just added company_id on budget for the usual record rule. And I created #34 (multi-company budget consolidation) for the future. It's still useful without it.